data-pipeline/src/data_pipeline/pipelines/variant_cooccurrence_counts.py

```diff
@@ -0,0 +1,152 @@
+import hail as hl
+
+TWO_HET_DATA_PATH = (
+    "gs://gnomadev-data-pipeline-output/pwd-2022-12-06/external_sources/chet_unphased_same_hap_per_gene.tsv"
+)
+HOMOZYGOUS_DATA_PATH = "gs://gnomadev-data-pipeline-output/pwd-2022-12-06/external_sources/het_hom_per_gene.tsv"
```

Comment:
If this is close to going to prod, maybe now is the time to discuss where we want to keep additional data sources that we may or may want to have in a bucket that we can maintain ownership of.

Currently, `/external_sources/` houses data like gencode, gtex, and clinvar. Those types of data feel like a different definition of external sources as opposed to things like variant cooccurence, genomic constraint, or other 'secondary analyses' type datasets.

Is it worth it to make a bucket to house secondary analyses type datasets so that we always at least have one (potentially outdated copy) of the input data? @mattsolo1, thoughts?

Reply:
I see what you're saying but in that case, seems like it would make a difference here that I got this data from Sarah and a pipeline that she maintains, rather than one under our purview. So to me that makes the TSV files feel like "primary" data, from our point of view at least. But I don't have a strong opinion on this point and am interested in @mattsolo1 's point of view.

Reply:
Ah, ok I think I wasn't very articulate. I bring this up 'cause it relates to the distinction both between Core Dataset and Secondary Analyses, and on our having a bucket to maintain control of data. So I think there are two points here to be addressed:

1. Do we want to have this data in a bucket that we control. Should we copy it to be also located somewhere in `gnomad-browser-data-pipeline`, in addition to the original place where we got it, or should we not copy it to our bucket at all and only reference the original source. I bring this up because I believe you had expressed interest in having our own copies of all input data.

2. If we want to copy it to be also in a bucket we control, which bucket is that. In this instance, going along with the downloads reformatting, there is a distinction between core dataset and secondary analyses, that distinction being who produces/maintains the dataset. If it is the methods team and thus the data is a part of a gnomad release, then it is considered core dataset. If not, it is a secondary analyses. Genomic Constraint, for example, is a dataset from Siwei. While its under some elses purview and produced from a pipeline they maintain, it is not part of the core gnomad dataset the methods team released, and considered 'secondary analyses' on the downloads page. I was curious if mirroring this structure could help keep the GCS bucket neat and tidy, as the current `external_sources` bucket seems to be distinct from this "datasets created using gnomad data", and this dataset also doesn't fit neatly in `gnomad_v2` or some bucket like that.

Reply:
> Is it worth it to make a bucket to house secondary analyses type datasets so that we always at least have one (potentially outdated copy) of the input data?

Yes I think so... and with some sort of version specifier -- maybe this is the date, would be ideal.  

 I'd be happy if we have separate folders that match the naming schema of downloads page.

So would the bucket schema is like `gs://gnomad-browser-data-pipeline/inputs/secondary-analyses/variant-coocurrence/<date>/<file>`?

Reply:
> So would the bucket schema is like gs://gnomad-browser-data-pipeline/inputs/secondary-analyses/variant-coocurrence/<date>/<file>?

Works for me--I like the idea of being more explicit about this now that I get what Riley meant.